utils/interfaces/schemas/library/telemetry/proxy/api/v2/apmtelemetry-request.json

```diff
@@ -605,7 +605,7 @@
 					"$ref": "#/$defs/Error"
 				}
 			},
-			"required": ["enabled"]
+			"$comment": "enabled should be a required field, but currently not implemented by python"
```

Comment:
You have a mechanism that allow ignoring some errors under condition, it's better than having a wrong schema for everybody. Feel free to add a meeting, we can do it together.